lib/plugins/index.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     serveStatic: require('./static'),
     throttle: require('./throttle'),
     urlEncodedBodyParser: require('./formBodyParser'),
+    inflightRequestThrottle: require('./inflightRequestThrottle'),
```

Comment:
We generally sort these alphabetically -- so please do that. And yes it'd be awesome if we had a lint rule for this.